index.html

```diff
@@ -61,34 +61,8 @@
       </p>
     </section>
     <section id="sotd">
-      <div class="warning">
-        <p>
-          Implementors need to be aware that the future work is now happening
-          on the <a href=
-          'https://w3c.github.io/geolocation-sensor/'>Geolocation Sensor</a>
-          specification in the <a href="https://www.w3.org/das/">Devices and
-          Sensors Working Group</a>.
-        </p>
-      </div>
-    </section>
-    <section id="conformance">
-      <p>
-        Requirements phrased in the imperative as part of algorithms (such as
-        "strip any leading space characters" or "return false and abort these
-        steps") are to be interpreted with the meaning of the key word ("MUST",
-        "SHOULD", "MAY", etc) used in introducing the algorithm.
-      </p>
       <p>
-        Conformance requirements phrased as algorithms or specific steps MAY be
-        implemented in any manner, so long as the end result is equivalent. (In
-        particular, the algorithms defined in this specification are intended
-        to be easy to follow, and not intended to be performant.)
-      </p>
-      <p id="hardwareLimitations">
-        User agents MAY impose implementation-specific limits on otherwise
-        unconstrained inputs, e.g. to prevent denial of service attacks, to
-        guard against running out of memory, or to work around
-        platform-specific limitations.
+        This specification is maintained.
```

Comment:
Probably want something more meaningful here....

Reply:
Since we still don't have Living Standards at W3C, we could simply quote the charter (s/Working Group/Devices and Sensors Working Group/) and clarify the group's positioning of these two APIs as follows:

```
The Devices and Sensors Working Group will maintain errata and new editions, as
necessary, for the Geolocation API. The Working Group is aware of <a
href="https://w3c.github.io/geolocation-sensor/#use-cases">more advanced use
cases</a> that cannot be realized using the Geolocation API. These use cases
are being addressed by the <a href="https://w3c.github.io/geolocation-sensor/">
Geolocation Sensor</a> specification.
```

Reply:
Since we still don't have Living Standards at W3C, we could simply quote the charter (s/Working Group/Devices and Sensors Working Group/) and clarify the group's positioning of these two APIs as follows:
```
The Devices and Sensors Working Group will maintain errata and new editions, as
necessary, for the Geolocation API. The Working Group is aware of <a
href="https://w3c.github.io/geolocation-sensor/#use-cases">more advanced use
cases</a> that cannot be realized using the Geolocation API. These use cases
are being addressed by the <a href="https://w3c.github.io/geolocation-sensor/">
Geolocation Sensor</a> specification.
```

Reply:
Great suggestion, ty! Will try to update this tomorrow.

Reply:
There is controversy around Geolocation Sensor being the place to deal with those use cases, so I'd feel better if we said nothing around that.